bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -34,9 +35,24 @@ spec:
         - |
           #!/bin/bash
           set -euo pipefail
-          chown -R openvswitch:openvswitch /var/run/openvswitch
-          chown -R openvswitch:openvswitch /etc/openvswitch
+          
+          export SYSTEMD_IGNORE_CHROOT=yes
+          # Check to see if ovs is provided by the node:
+          if systemctl status ovsdb-server; then
```

Comment:
systemctl status won't return 0 if the service is stopped.
Shouldn't we do something that returns 1 if the unit is available to be executed even if it's stopped? For instance systemctl cat ovsdb-server

Reply:
We should not be starting openvswitch. We should just handle the case where it is started.